src/openms/source/TRANSFORMATIONS/FEATUREFINDER/EmgFitter1D.cpp

```diff
@@ -265,15 +269,20 @@ namespace OpenMS
     if (boost::math::isinf(symmetry_) || boost::math::isnan(symmetry_))
     {
       symmetric_ = true;
-      symmetry_ = 10;
+      symmetry_ = 10.0;
     }
 
     // optimize the symmetry
     // The computations can lead to an overflow error at very low values of symmetry (s~0).
     // For s~5 the parameter can be approximated by the Levenberg-Marquardt algorithms.
     // (the other parameters are much greater than one)
     if (symmetry_ < 1)
+    {
       symmetry_ += 5;
+    }
+
+    // Need to ensure that we do not go beyond the maximal width of the peak
+    symmetry_ = std::min(symmetry_, max_peak_width);
```

Comment:
this is especially a problem when we are measuring time in minutes and not in seconds